misc/org.projecthamster.hamster-gtk.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=hamster-gtk
```

Comment:
For now I would like to be as specific as possible. And whilst the name certainly is not set in stone, it should not be too generic there may be several "hamster time tracker" frontends in future and we should give the impression that the GTK one is somehow special.